docker-compose.yml

```diff
@@ -18,75 +18,76 @@ services:
         - APPLICANT_FRONTEND_HOST=http://localhost:3008
+        - RSA256_PRIVATE_KEY_BASE64=LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlDV3dJQkFBS0JnUUNvbEdzV0FiVzlWUG40Zm9URW8vdXdZRXpBUElDQkRZREc2ZlRacG11VStKclVmZkF6Cjl0aGkxRzQyS0NEQUFsbFMySmRFaUVlMDZzaXpyeURqV3liQnYycXZKTEtpNFlWdEl4b3RjN1lGdStmM0htUFMKMlo4SVNMdCtMVTBjb2JKQVZMREpIekhZWk9idW1jUTFhbEcvTy83TFJHdFQ1clNlTkJZUCtkOUs5d0lEQVFBQgpBb0dBSVZqYXBndklpNTlYbzFSRHB6SlpBMWJKVisyZjdlZGg1ZmM1dkNRemU5Q0JzQzhKYVA0WFo5Vmx4OGpBClVic2xPRUlvQ2lKN0xNcVUwUlovaHZucy9HeEJLdjNZWFI4TjFkY3JmdUVPVm5VK0FsSERUSjdZT2tWZUloeGoKcDAyZnUyNmFKeFpSWDVvZjBCb004U2xYMUM2clpoSVBUek1kTzRpNlhIVnhxZUVDUVFEWTVpMWdaZTZHMm5VZgo3V3ZpWG13SHZxMEh0cWgzRDlVK3UrYUR1Nk1BNjJlQ3hhaWVaYUpQSitsSWtwRGVpU2VOOUVWMWJaVTJNdy9uCnR4cm51OG9qQWtFQXh2aFJjaFg4ZTA5N3FsMndMZk9JUVhZblFtZGZRN3pBM3JSOE5DaEU2dmNxcnpZaDFpeXkKVjhxcXdwbWJRbUphR1ZDbVBtcDhoMWhGbHpiTzJETFhIUUpBQzU2eE9hUGNWbjhublRGRUFWbWJvZ0tXRkg2MQpvY05JWWxjQXIzZ0M4STZyK2ZhSzVRbCsyNHpBMDdGK2IxMmJyK0F4Ukd3OWZTelpJa25OK0RJNFR3SkFKTFlVCmNQUlZqWHJ3MmxKRyszTi9wSFJrY2dBWXQ2clArMGRaMDFYS3dYK3hSSHE4YTRQN3MrOFU2cXJrMXE3RzZQelUKYjVlK09rcXpMRjlVY3RlT2tRSkFlSzZXL0JtMDR5L2ZnSis3VWlIeldzdnozOWdaRFZnblBBQzB3ZmdMTUF2TApna0NjQkttWW9CM2tYUitGSlNlNVZFNlBSRnBjVmc4bmwxWSswNlZCdlE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQ==
+        - RSA256_PUBLIC_KEY_BASE64=LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlHZk1BMEdDU3FHU0liM0RRRUJBUVVBQTRHTkFEQ0JpUUtCZ1FDb2xHc1dBYlc5VlBuNGZvVEVvL3V3WUV6QQpQSUNCRFlERzZmVFpwbXVVK0pyVWZmQXo5dGhpMUc0MktDREFBbGxTMkpkRWlFZTA2c2l6cnlEald5YkJ2MnF2CkpMS2k0WVZ0SXhvdGM3WUZ1K2YzSG1QUzJaOElTTHQrTFUwY29iSkFWTERKSHpIWVpPYnVtY1ExYWxHL08vN0wKUkd0VDVyU2VOQllQK2Q5Szl3SURBUUFCCi0tLS0tRU5EIFBVQkxJQyBLRVktLS0tLQ==
+        - FLASK_ENV=development
   assessment-store:
-      build: 
+      build:
         context: ../funding-service-design-assessment-store
       volumes: ['../funding-service-design-assessment-store:/app']
       command: bash -c "flask db upgrade && flask run -p 8080 -h 0.0.0.0"
-      ports: 
+      ports:
         - 3005:8080
       environment:
         - DATABASE_URL=postgresql://postgres:password@database:5432/assessment_store
-      depends_on: 
+      depends_on:
         - database
   notification:
-      build: 
+      build:
         context: ../funding-service-design-notification
       volumes: ['../funding-service-design-notification:/app']
       ports:
         - 3006:8080
       environment:
         GOV_NOTIFY_API_KEY: ${GOV_NOTIFY_API_KEY:?err}
-  fund-discovery-frontend:
-    build: 
-      context: ../funding-service-design-fund-discovery-frontend
-    volumes: ['../funding-service-design-fund-discovery-frontend:/app']
-    command: ['flask', 'run', '--host', '0.0.0.0', '--port', '8080']
-    depends_on:
-      - fund-store
-    environment:
-      - FLASK_ENV=development
-      - AUTHENTICATOR_HOST=http://localhost:3004
-      - FUND_STORE_API_HOST=http://fund-store:8080
-      - USE_LOCAL_DATA=False
-    ports: 
-      - 3007:8080
+#  fund-discovery-frontend:
```

Comment:
Why are we commenting this out? Seems unrelated to this PR?

Reply:
I'd commented this locally as fund-discovery hasn't been in use in the wider service, but not part of this PR as you say, so reinstated 👍